### ISSUE TITLE
3/24 진행상황

### ISSUE THREAD

**Issue by Doyun-lab** [Doyun-lab/MACE_classification#25]
- Raw Data를 Down-sampling 하여 DNN 진행했을 때, 균형적으로 성능이 잘 나오는 것을 확인- Raw Data를 Down-sampling 하여 DNN 진행했을 때, 균형적으로 성능이 잘 나오는 것을 확인  

<br>

- DNN은 Stopping Condition을 잡아야 하기때문에 Validation Data 필요 (Validation Data 기준으로 Stop, Best model 결정)
- 모든 Data가 1번은 Test 되어야 하므로 10-fold로 Train + Validation / Test를 9:1 비율로 나누기
- Train + Validation set 에서만 Down-sampling 되도록 Model 생성한 후, Test set로 성능 측정
> 먼저 Down-sampling을 진행하게 되면 Test set도 Down-sampling 되는 것이기 때문에 Train + Validation / Test 로 나눈 후 Train + Validation 만 Down-sampling 할 방법 강구할 필요

<br>

> Test는 실제 환경이기 때문에 실제 환경에서 Sample이 Down 되어 들어오지 않아 Test는 Down-sampling 실시하면 안됨
